src/services/escola-lms/vouchers.ts

```diff
@@ -0,0 +1,70 @@
+import { request } from 'umi';
```

Comment:
use requests from export https://escolalms.github.io/ts-models/modules/EscolaLms.Vouchers.Http.Requests.html

Reply:
Add this defs to all requests